modules/nf-core/bedtools/intersect/main.nf

```diff
@@ -10,6 +10,7 @@ process BEDTOOLS_INTERSECT {
     input:
     tuple val(meta), path(intervals1), path(intervals2)
     val extension
```

Comment:
Can you remove this `extension` value and work with `task.ext.suffix` instead?

Reply:
done